productinfo/ec2/productinfo_ec2_test.go

```diff
@@ -143,10 +144,67 @@ func (dps *DummyPricingSource) GetProducts(input *pricing.GetProductsInput) (*pr
 }
 
 // strPointer gets the pointer to the passed string
-func (dps *DummyPricingSource) strPointer(str string) *string {
+func (dps *dummy) strPointer(str string) *string {
 	return &str
 }
 
+func (dps *dummy) DescribeAvailabilityZones(input *ec2.DescribeAvailabilityZonesInput) (*ec2.DescribeAvailabilityZonesOutput, error) {
+	if dps.TcId == 10 {
+		return nil, errors.New("could not get information about zones")
+	}
+	return &ec2.DescribeAvailabilityZonesOutput{
+		AvailabilityZones: []*ec2.AvailabilityZone{
+			{
+				State:      aws.String("available"),
```

Comment:
please check if there are constants in the ec2 library for the strings here

Reply:
thx